monitor/modules/src/main/resources/modules/org/wildfly/swarm/monitor/main/module.xml

```diff
@@ -2,7 +2,7 @@
 
   <dependencies>
     <!-- For when run with bonafide IDE classpath -->
-    <system export="true">
+    <system export="false">    <!-- this needs to be suppressed, otherwise we classloading problems in the jaxrs tests-->
```

Comment:
yes, but it was the only way I could get it working. i'll catch you on irc to discuss the dependencies and how they can be aligned ti improve the situation.